lib/main.dart

```diff
@@ -1,113 +1,67 @@
-import 'package:flutter/material.dart';
+import "package:flutter/material.dart";
+
+import "./screens/settings_screen.dart";
+import "./screens/sub_chp_screen.dart";
+import "./screens/sub_screen.dart";
 
 void main() {
   runApp(MyApp());
 }
 
+/// Root Widget of the App
 class MyApp extends StatelessWidget {
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
+      debugShowCheckedModeBanner: false,
+      title: "Bookify",
       theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
+        textTheme: const TextTheme(
+          headline6: TextStyle(
+            fontFamily: "Montserrat",
+            fontWeight: FontWeight.w700,
+            fontSize: 40,
+          ),
+          headline5: TextStyle(
+            fontFamily: "Montserrat",
+            fontWeight: FontWeight.w600,
+            fontSize: 22.5,
+          ),
+          button: TextStyle(
+            fontFamily: "SF UI Display",
+            fontWeight: FontWeight.w500,
+            fontSize: 22.5,
+          ),
+          bodyText1: TextStyle(
+            fontFamily: "Montserrat",
+            fontWeight: FontWeight.w500,
+            fontSize: 19.5,
+          ),
+        ),
+        primarySwatch: Colors.orange,
       ),
-      home: MyHomePage(title: 'Flutter Demo Home Page'),
+      home: const MyHomePage(),
+      routes: <String, WidgetBuilder>{
+        SubScreen.routename: (BuildContext ctx) => const SubScreen(),
+        SubChpScreen.routename: (BuildContext ctx) => const SubChpScreen(),
+        SettingsScreen.routename: (BuildContext ctx) => SettingsScreen(),
```

Comment:
no `settings_screen.dart`  file created

Reply:
I am building that. It will complete soon you may merge till then.

Reply:
Okay! Your PR will be merged at least comment out those two lines to remove errors.

Reply:
So @Mahi-Badaya Should I merge the Request.

Reply:
No, let him remove the error first.

Reply:
@Megabyte-143  do this task asap.

Reply:
done